gmt/datasets/tutorial.py

```diff
@@ -38,6 +38,27 @@ def load_japan_quakes():
     return data
 
 
+def load_tut_ship():
+    """
+    Load a table of ship observations of bathymetry off Baja California as a
+    pandas.DataFrame.
+
+    This is the ``@tut_ship.xyz`` dataset used in the GMT tutorials.
+
+    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
+    first time you invoke this function. Afterwards, it will load the data from
+    the cache. So you'll need an internet connection the first time around.
+
+    Returns
+    -------
+    data :  pandas.Dataframe
+        The data table. Columns are x, y, and z.
+    """
+    fname = which("@tut_ship.xyz", download="c")
+    data = pd.read_csv(fname, sep="\t", header=None, names=["x", "y", "z"])
```

Comment:
I always prefer using longer explicit names to avoid confusion. In this case, avoid using `x, y, z`. The coordinates are actually longitude, latitude, and bathymetry.

```suggestion
    data = pd.read_csv(fname, sep="\t", header=None, names=["longitude", "latitude", "bathymetry"])
```

Reply:
Ok, I've made the change and noted in the commit message that the sample dataset actually has longitude in the 0-360 format.